### PR TITLE
Add gsw

### PR DESCRIPTION
**Pull request by karpfen**

### CODE REVIEW

**Comment by goergen95** [comment]
Hi! Why removing the years parameters? It might also be a good idea to include it in the get function. Currently one would always download and calculate the complete time series?

**Comment by karpfen** [comment]
@goergen95 I removed them only from the calc function. The user can still select which years to process while setting up the portfolio.

**Comment by goergen95** [comment]
Ah, yep! Sorry, did not catch that. 👍

**Comment by karpfen** [comment]
Now that I looked into it, I noticed that we're not really consistent within the package with that logic, though.
For example, in human footprint you also set the year during get_*, but for GFW you do it during calc_*.

**Comment by goergen95** [comment]
GFW are single layers with no temporal dimension. The temporal information is in the pixel's value, not the layers. However, I would like to see it consistent with other resources/indicators, aka in relation to humanfootprint. One could argue that one might be downloading all available layers in one sweep, but for certain analysis one might only calculate these indicators for a temporal subset. That would mean to expose the years argument in both functions.

File: data-raw/gsw_time_series.R [hunk 1]
```diff
@@ -0,0 +1,31 @@
+library(dplyr)
+library(mapme.indicators)
+library(terra)
+
+years <- 2000:2001
+outdir <- system.file("resources", package = "mapme.indicators")
```
Comment (goergen95) [inline data-raw/gsw_time_series.R:6]: This needs changing to reference mapme.biodiversity

File: data-raw/gsw_time_series.R [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  system.file("extdata", "shell_beach_protected_area_41057_B.gpkg",
+              package = "mapme.biodiversity"
+  )
+) %>%
+get_resources(get_gsw_time_series(years = years))
+
+gsw_time_series <- prep_resources(x)
+gsw_time_series <- gsw_time_series [[1]]
+
+outdir <- "inst/resources/gsw_time_series"
```
Comment (goergen95) [inline data-raw/gsw_time_series.R:19]: This should reference 'res' instead of 'resources'

**Comment by karpfen** [comment]
Ah, true. That makes sense then.

File: tests/testthat/test-get_gsw_time_series.R [hunk 3]
```diff
@@ -0,0 +1,34 @@
+test_that("get_gsw_time_series works", {
```
Comment (goergen95) [inline tests/testthat/test-get_gsw_time_series.R:1]: Could this include a test to see that a footprints object is returned if successful?

**Review by goergen95 — CHANGES_REQUESTED** [review]

**Review by goergen95 — APPROVED** [review]

**Comment by goergen95** [comment]
@karpfen: the code in the example fails

**Pull request merged**